lib/resources/collection/index.js

```diff
@@ -448,7 +448,7 @@ Collection.prototype.save = function (ctx, fn) {
 
   domain.changed =  function (property) {
```

Comment:
I go for OR case. 

I would like to know anyone will go for AND case and reason for that.